jboss-eap-standalone-rhel7/azuredeploy.parameters.json

```diff
@@ -3,28 +3,31 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "adminUsername": {
-          "value": "GEN-UNIQUE"
+          "value": "changeme"
```

Comment:
revert all of these changes in the parameters file

Reply:
Reverted parameters' values except for rhsmUserName, rhsmPassword, rhsmPool because those parameters should be input by the user's data.